keps/693-multikueue/README.md

```diff
@@ -205,7 +217,15 @@ type KubeConfig struct {
     LocationType LocationType `json:"locationType"`
 }
 
-type MultiKueueClusterStatus {
+type ClusterProfile struct {
```

Comment:
How do we know for this ClusterProfile which CredentialProvider to use as defined in the ConfigAPI?